src/IBM.Watson.CompareComply.v1/README.md

```diff
@@ -1,216 +0,0 @@
-[![NuGet](https://img.shields.io/badge/nuget-v2.16.0-green.svg?style=flat)](https://www.nuget.org/packages/IBM.Watson.CompareComply.v1/)
```

Comment:
Did you mean to delete the READMEs?

Reply:
Yes, these were all handwritten. I will create generate versions for each service in the future.